src/main/java/com/google/sps/servlets/QueryFactory.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets; 
+
+import com.google.cloud.spanner.Statement;
+import java.util.List;
+
+final class QueryFactory {
+  
+  private static final QueryFactory instance = new QueryFactory();
+  private static final String GET_COLUMNS_FROM_TABLES = "SELECT table_name, ARRAY_AGG(column_name) FROM information_schema.columns WHERE table_name in(";
+  private static final String GROUP_BY_TABLE_NAMES = ") group by table_name";
+  private static final String SCHEMA_INFO_SQL = "SELECT column_name, spanner_type, is_nullable FROM information_schema.columns WHERE table_name = '";
+
+  private QueryFactory() {
+    // Private because static class
+  }
+
+  static QueryFactory getInstance() {
+    return instance;
+  }
+
+  static String buildSchemaQuery(String table) {
```

Comment:
Sometimes we return Strings, other times Statements. Since we are going to need to bind variables to the queries, it'll be a good idea to consolidate these into returning Statement rather than String.

Reply:
Done - made them all into Statements